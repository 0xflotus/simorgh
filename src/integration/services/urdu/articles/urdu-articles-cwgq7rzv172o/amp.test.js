/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service urdu
 * @pathname /urdu/articles/cwgq7rzv172o
 */

import runAmpTests from '../../../../pages/articles/ampTests';

describe('AMP urdu articles', runAmpTests);
