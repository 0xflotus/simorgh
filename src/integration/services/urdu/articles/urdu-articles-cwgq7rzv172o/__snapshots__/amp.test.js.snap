// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP urdu articles A11y I can see the skip to content link 1`] = `"مواد پر جائیں"`;

exports[`AMP urdu articles AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP urdu articles AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP urdu articles Analytics ATI 1`] = `
Object {
  "requests": Object {
    "base": "https://logws1363.ati-host.net?",
    "pageview": "\${base}s=598343&s2=95&p=urdu.articles.cwgq7rzv172o.page&r=\${screenWidth}x\${screenHeight}x\${screenColorDepth}&re=\${availableScreenWidth}x\${availableScreenHeight}&hl=\${timestamp}&lng=\${browserLanguage}&x1=[urn:bbc:optimo:cwgq7rzv172o]&x2=[amp]&x3=[news-urdu]&x4=[ur]&x5=[\${sourceUrl}]&x6=[\${documentReferrer}]&x7=[article]&x8=[simorgh]&x9=[یہ+اس+ٹیسٹ+مضمون+کی+سرخی+ہے۔]&x11=[2019-10-04T10:44:53.338Z]&x12=[2019-10-04T10:44:53.338Z]&ref=\${documentReferrer}",
  },
  "transport": Object {
    "beacon": false,
    "image": true,
    "xhrpost": false,
  },
  "triggers": Object {
    "trackPageview": Object {
      "on": "visible",
      "request": "pageview",
    },
  },
}
`;

exports[`AMP urdu articles Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/urdu"`;

exports[`AMP urdu articles Footer I can see the footer copyright and external linking text 1`] = `"© 2020 بی بی سی. بی بی سی بیرونی ویب سائٹس کے مواد کا ذمہ دار نہیں بیرونی لنکس کے بارے میں ہماری پالیسی."`;

exports[`AMP urdu articles Header I can see the branding 1`] = `"BBC News, اردو"`;

exports[`AMP urdu articles I can see an image with a caption 1`] = `"https://ichef.bbci.co.uk/news/640/cpsdevpb/d1c3/test/74f82030-e693-11e9-88d6-81aa2ccc6d4c.jpg"`;

exports[`AMP urdu articles I can see an image with a caption 2`] = `"،تصویر کا کیپشنکاپی رائٹ بی بی سی کی اس آزمائشی تصویر میں فرانس کا نقشہ دکھایا گیا ہے۔ شبیہہ پہلے تین بلاکس میں ہے اور اس کی سرخی ہے۔"`;

exports[`AMP urdu articles SEO Canonical link 1`] = `"http://localhost:7080/urdu/articles/cwgq7rzv172o"`;

exports[`AMP urdu articles SEO Dir attribute 1`] = `"rtl"`;

exports[`AMP urdu articles SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP urdu articles SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP urdu articles SEO Heading level 1 1`] = `"یہ اس ٹیسٹ مضمون کی سرخی ہے۔"`;

exports[`AMP urdu articles SEO Lang attribute 1`] = `"ur"`;

exports[`AMP urdu articles SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/urdu.png",
      "width": 1024,
    },
    "name": "BBC News اردو",
    "noBylinesPolicy": "https://www.bbc.com/urdu/institutional-49282935#authorexpertise",
  },
  "dateModified": "2019-10-04T10:44:53.338Z",
  "datePublished": "2019-10-04T10:44:53.338Z",
  "headline": "یہ اس ٹیسٹ مضمون کی سرخی ہے۔",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/urdu.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/urdu/articles/cwgq7rzv172o",
    "@type": "WebPage",
    "name": "یہ اس ٹیسٹ مضمون کی سرخی ہے۔",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/urdu.png",
      "width": 1024,
    },
    "name": "BBC News اردو",
    "publishingPrinciples": "https://www.bbc.com/urdu/institutional-49282935",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/urdu.png",
  "url": "http://localhost:7080/urdu/articles/cwgq7rzv172o",
}
`;

exports[`AMP urdu articles SEO OG description 1`] = `"یہ اس ٹیسٹ مضمون کی سرخی ہے۔"`;

exports[`AMP urdu articles SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/urdu.png"`;

exports[`AMP urdu articles SEO OG image alt 1`] = `"BBC News اردو"`;

exports[`AMP urdu articles SEO OG locale 1`] = `"ur"`;

exports[`AMP urdu articles SEO OG site name 1`] = `"BBC News اردو"`;

exports[`AMP urdu articles SEO OG title 1`] = `"یہ اس ٹیسٹ مضمون کی سرخی ہے۔ - BBC News اردو"`;

exports[`AMP urdu articles SEO OG type 1`] = `"article"`;

exports[`AMP urdu articles SEO OG url 1`] = `"http://localhost:7080/urdu/articles/cwgq7rzv172o"`;

exports[`AMP urdu articles SEO Page title 1`] = `"یہ اس ٹیسٹ مضمون کی سرخی ہے۔ - BBC News اردو"`;

exports[`AMP urdu articles SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP urdu articles SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP urdu articles SEO Twitter creator 1`] = `"@bbcurdu"`;

exports[`AMP urdu articles SEO Twitter description 1`] = `"یہ اس ٹیسٹ مضمون کی سرخی ہے۔"`;

exports[`AMP urdu articles SEO Twitter image alt 1`] = `"BBC News اردو"`;

exports[`AMP urdu articles SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/urdu.png"`;

exports[`AMP urdu articles SEO Twitter site 1`] = `"@bbcurdu"`;

exports[`AMP urdu articles SEO Twitter title 1`] = `"یہ اس ٹیسٹ مضمون کی سرخی ہے۔ - BBC News اردو"`;
